lib/tests/systems.nix

```diff
@@ -10,7 +10,7 @@ let
     expected = lib.sort lib.lessThan y;
   };
 
-  /*
+  /**
```

Comment:
This whole comment should probably turn back into a normal comment, it's for a function that is not exposed, and used only as a local helper.